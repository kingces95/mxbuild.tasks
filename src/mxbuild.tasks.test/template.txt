Items0:
Id=%(Items0.Identity), ElemA=%(ElementA), ElemB=%(ElementB)

Items1:
Id=%(Items1.Identity), ElemA=%(ElementA), ElemB=%(ElementB)

Items:2
Id=%(Items2.Identity), ElemA=%(ElementA), ElemB=%(ElementB)