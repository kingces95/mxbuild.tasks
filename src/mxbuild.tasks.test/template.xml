<?xml version="1.0" encoding="utf-8"?>
<root>
  <group id="%(Items0.Identity)">
    <element>{'%(ElementA)' '%(ElementB)'}</element>
    <cdata><![CDATA[%(Cdata)]]></cdata>
    <attribute attribute="%(Attribute)" />
  </group>
  <group id="%(Items1.Identity)" />
  <group id="%(Items2.Identity)" />
</root>