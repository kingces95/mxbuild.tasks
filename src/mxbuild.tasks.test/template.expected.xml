<?xml version="1.0" encoding="utf-8"?>
<root>
  <group id="identity 0.0">
    <element>{'elementA 0.0' 'elementB 0.0'}</element>
    <cdata><![CDATA[cdata 0.0]]></cdata>
    <attribute attribute="attribute 0.0" />
  </group>
  <group id="identity 0.1">
    <element>{'elementA 0.1' 'elementB 0.1'}</element>
    <cdata><![CDATA[cdata 0.1]]></cdata>
    <attribute attribute="attribute 0.1" />
  </group>
  <group id="identity 1.0" />
  <group id="identity 1.1" />
</root>